versions.properties

```diff
@@ -19,6 +19,8 @@ plugin.android=4.1.0-rc02
 
 plugin.com.osacky.doctor=0.5.1
 
+plugin.com.gradleup.auto.manifest=1.0.3
+
```

Comment:
I'd prefer to keep these alphabetically ordered for straightforward lookup and consistency (above `plugin.com.osacky.doctor` in this case).